### ISSUE TITLE
Recall keeps zero

### ISSUE THREAD

**Issue by Yuantian013** [experiencor/keras-yolo2#335]
Hi,
I use the official raccoon_dataset-master data as my dataset, and train with the tiny yolo, but the current recall will keep in 0 even during 10 more epochs.I can't find out the problem.
This is my json:
{
    "model" : {
        "backend":              "Tiny Yolo",
        "input_size":           200,
        "anchors":              [1.94,2.51, 2.69,4.55, 3.62,5.55, 4.77,4.23, 5.31,5.78],
        "max_box_per_image":    10,
        "labels":               ["raccoon"]
    },

    "train": {
        "train_image_folder":   "/Users/tianyuan/Desktop/keras-yolo2-master/raccoon_dataset-master/images/",
        "train_annot_folder":   "/Users/tianyuan/Desktop/keras-yolo2-master/raccoon_dataset-master/annotations/",     
          
        "train_times":          1,
        "pretrained_weights":   "",
        "batch_size":           16,
        "learning_rate":        1e-4,
        "nb_epochs":            50,
        "warmup_epochs":        3,

        "object_scale":         5.0 ,
        "no_object_scale":      0.5,
        "coord_scale":          2.0,
        "class_scale":          1.0,

        "saved_weights_name":   "/Users/tianyuan/Desktop/keras-yolo2-master/test_weight.h5",
        "debug":                true
    },

    "valid": {
        "valid_image_folder":   "",
        "valid_annot_folder":   "",

        "valid_times":          1
    }
}

**Comment by neney1234** [comment]
Same problem

**Comment by NPetsky** [comment]
I see from your config that you are in the warmup mode I think then recall can be zero (this is not the real training) see #117 #164 and #293 
Set warmup_epochs to zero and pretrained_weights to those you have now saved (if you have not quit the warmup training) then you should start training and after some iterations recall should be non zero
Hope I could help!

**Comment by letilessa** [comment]
Hi @Yuantian013 and @neney1234, did this answer work for you? I tried but I am still getting recall zero.
